pkg/arvo/app/metadata-pull-hook.hoon

```diff
@@ -149,7 +195,11 @@
   =+  !<(=hook-update:metadata vase)
   ?.  ?=(%preview -.hook-update)
     (on-poke:def mark vase)
-  :_  this(previews (~(put by previews) group.hook-update +.hook-update))
+  =.  previews
```

Comment:
=: is more appropriate for these two modifications than =.

Reply:
good catch!